api/src/main/java/com/velocitypowered/api/proxy/ServerConnection.java

```diff
@@ -37,4 +39,12 @@
    * @return the player for this connection
    */
   Player getPlayer();
+
+  /**
+   * Returns the world the current connection's associated {@link Player} is playing on.
+   *
+   * @return an {@link Optional} the name of the world that the player is playing on, which may be
+   *     empty.
+   */
+  Optional<Key> getCurrentWorldName();
```

Comment:
I would also document the optional is empty if the client's version is <1.16 (I'm not too sure that's the right version, going with what the linked issue says).

Reply:
that's what I questioned; whether to document that it will be an empty optional if the version is < 1.16